src/monocle_apptrace/wrap_common.py

```diff
@@ -164,7 +164,8 @@ def process_span(to_wrap, span, instance, args):
                         if attribute and accessor:
                             attribute_name = f"entity.{span_index}.{attribute}"
                             try:
-                                result = eval(accessor)(instance, args)
+                                arguments = {"instance":instance, "args":args, "kwargs":kwargs, "return_value":return_value}
```

Comment:
instead of 'return_value' can it be 'result' or 'output'

Reply:
also, document what goes in arguments in [monocle_user_guide.md](https://github.com/monocle2ai/monocle/blob/main/Monocle_User_Guide.md)